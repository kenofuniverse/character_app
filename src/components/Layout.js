import React from 'react'
import PropTypes from 'prop-types'
import { Container } from 'react-bootstrap'
import Header from './Header'
// import Footer from './Footer'

const Layout = ({ children }) => (
  <Container fluid={true}>
    <Header />
    <Container>
      {children}
    </Container>
    {/* <Footer /> */}
  </Container>
)

Layout.propTypes = {
  children: PropTypes.object,
}

export default Layout
