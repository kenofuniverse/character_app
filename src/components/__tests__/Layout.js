import React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router';
import Layout from '../Layout'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Layout />
      </MemoryRouter>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
