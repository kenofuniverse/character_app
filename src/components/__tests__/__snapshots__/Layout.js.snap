// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="layout"
>
  <nav
    className="navbar navbar-expand navbar-dark bg-dark"
  >
    <div
      className="container"
    >
      <a
        className="navbar-brand"
        href="/"
        onClick={[Function]}
      >
        Milo Credit Test
      </a>
      <div
        className="mr-auto navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rb-event-key={null}
          disabled={false}
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </div>
      <form
        className="form-inline"
        onSubmit={[Function]}
      >
        <input
          className="mr-sm-2 form-control"
          onChange={[Function]}
          placeholder="Search Chracters"
          type="text"
          value=""
        />
        <button
          className="btn btn-primary"
          disabled={false}
          type="submit"
        >
          Search
        </button>
      </form>
    </div>
  </nav>
  <div
    className="layout__container container"
  />
</div>
`;
